integration/csharp/tests/Duckstax.Otterbrix.Tests/WrapperTest.cs

```diff
@@ -0,0 +1,286 @@
+            Assert.IsTrue(cursor.Size() == 100);
+
+            {
+                DocumentWrapper doc = cursor.Next();
+                Assert.IsTrue(doc != null);
+                Assert.IsTrue(doc.GetLong("count") == 0);
+                Assert.IsTrue(doc.GetString("name") == "Name 0");
+            }
+            {
+                DocumentWrapper doc = cursor.Next();
+                Assert.IsTrue(doc != null);
+                Assert.IsTrue(doc.GetLong("count") == 1);
+                Assert.IsTrue(doc.GetString("name") == "Name 1");
+            }
+            {
+                DocumentWrapper doc = cursor.Next();
+                Assert.IsTrue(doc != null);
+                Assert.IsTrue(doc.GetLong("count") == 2);
+                Assert.IsTrue(doc.GetString("name") == "Name 2");
+            }
+            {
+                DocumentWrapper doc = cursor.Next();
+                Assert.IsTrue(doc != null);
+                Assert.IsTrue(doc.GetLong("count") == 3);
+                Assert.IsTrue(doc.GetString("name") == "Name 3");
+            }
+            {
+                DocumentWrapper doc = cursor.Next();
+                Assert.IsTrue(doc != null);
+                Assert.IsTrue(doc.GetLong("count") == 4);
+                Assert.IsTrue(doc.GetString("name") == "Name 4");
+            }
+        }
+        {
+            string query = "SELECT * FROM TestDatabase.TestCollection ORDER BY count DESC;";
+            CursorWrapper cursor = otterbrix.Execute(query);
+            Assert.IsTrue(cursor != null);
+            Assert.IsTrue(cursor.IsSuccess());
+            Assert.IsFalse(cursor.IsError());
+            Assert.IsTrue(cursor.Size() == 100);
+
+            {
+                DocumentWrapper doc = cursor.Next();
+                Assert.IsTrue(doc != null);
+                Assert.IsTrue(doc.GetLong("count") == 99);
+                Assert.IsTrue(doc.GetString("name") == "Name 99");
+            }
+            {
+                DocumentWrapper doc = cursor.Next();
+                Assert.IsTrue(doc != null);
+                Assert.IsTrue(doc.GetLong("count") == 98);
+                Assert.IsTrue(doc.GetString("name") == "Name 98");
+            }
+            {
+                DocumentWrapper doc = cursor.Next();
+                Assert.IsTrue(doc != null);
+                Assert.IsTrue(doc.GetLong("count") == 97);
+                Assert.IsTrue(doc.GetString("name") == "Name 97");
+            }
+            {
+                DocumentWrapper doc = cursor.Next();
+                Assert.IsTrue(doc != null);
+                Assert.IsTrue(doc.GetLong("count") == 96);
+                Assert.IsTrue(doc.GetString("name") == "Name 96");
+            }
+            {
+                DocumentWrapper doc = cursor.Next();
+                Assert.IsTrue(doc != null);
+                Assert.IsTrue(doc.GetLong("count") == 95);
+                Assert.IsTrue(doc.GetString("name") == "Name 95");
+            }
```

Comment:
How about a `for` loop? Possibly You can end with [TestCase parameters](https://www.lambdatest.com/blog/nunit-parameterized-test-examples/) - it's easier to read and maintain.